app/src/main/java/com/vanniktech/emoji/sample/MainActivity.java

```diff
@@ -30,6 +30,7 @@
 import com.vanniktech.emoji.one.EmojiOneProvider;
 import com.vanniktech.emoji.twitter.TwitterEmojiProvider;
 
+@SuppressWarnings("CPD-START") // We don't care about duplicate code in the sample.
```

Comment:
Found here: https://stackoverflow.com/a/18625921/4279995
Not optimal, but I think there is no better way...